selectors/actions_menu.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {GlobalState} from 'types/store';
+import {get} from 'mattermost-redux/selectors/entities/preferences';
+import {Preferences} from 'mattermost-redux/constants';
+
+export function showActionsMenuPulsatingDotPulsatingDot(state: GlobalState): boolean {
```

Comment:
Is the double PulsatingDot in the naming on purpose or by mistake? 😄

Reply:
Thanks for catching this one! :) That's an embarrassing function name

Reply:
fixed ec95b79